tests/h/search/query_test.py

```diff
@@ -415,15 +415,6 @@ def test_filters_on_whole_url(self, Annotation, get_search):
 
         assert sorted(result.annotation_ids) == sorted(expected_ids)
 
-    def test_filter_matches_invalid_uri(self, Annotation, get_search):
```

Comment:
Per our discussion that this is no longer expected behavior as the scheme will be auto added and the uri will become valid, I've removed this test.